### PR TITLE
[DOCU-3685]: WIP Install page

### PR DESCRIPTION
**Pull request by fabianrbz**
### Testing instructions

Preview link: https://deploy-preview-7030--kongdocs.netlify.app/install/

### Checklist 

- [ ] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@fabianrbz I opened a draft PR that adds the install pages back: https://github.com/kumahq/kuma-website/pull/1682. I was having problems viewing it locally, so I'm just going to check that everything works correctly before asking the Mesh team to approve and merge. I'm hoping I can get some predicted URLs to you soon once I'm able to view the preview docs.

**Comment by cloudjumpercat** [comment]
@fabianrbz 
**Kuma predicted URLs:**

- Kubernetes: https://kuma.io/docs/latest/installation/kubernetes/
- Helm: https://kuma.io/docs/latest/installation/helm/
- OpenShift: https://kuma.io/docs/latest/installation/openshift/
- Docker: https://kuma.io/docs/latest/installation/docker/
- Amazon Linux: https://kuma.io/docs/latest/installation/amazonlinux/
- Amazon EKS: https://kuma.io/docs/latest/installation/kubernetes/
- CentOS: https://kuma.io/docs/latest/installation/centos/
- RedHat: https://kuma.io/docs/latest/installation/redhat/
- Debian: https://kuma.io/docs/latest/installation/debian/
- Ubuntu: https://kuma.io/docs/latest/installation/ubuntu/

**Kong Mesh predicted URLs:**

- Kubernetes: https://docs.konghq.com/mesh/latest/installation/kubernetes/
- Helm: https://docs.konghq.com/mesh/latest/installation/helm/
- OpenShift: https://docs.konghq.com/mesh/latest/installation/openshift/
- Docker: https://docs.konghq.com/mesh/latest/installation/docker/
- Amazon Linux: https://docs.konghq.com/mesh/latest/installation/amazonlinux/
- Amazon EKS: https://docs.konghq.com/mesh/latest/installation/kubernetes/
- CentOS: https://docs.konghq.com/mesh/latest/installation/centos/
- RedHat: https://docs.konghq.com/mesh/latest/installation/redhat/
- Debian: https://docs.konghq.com/mesh/latest/installation/debian/
- Ubuntu: https://docs.konghq.com/mesh/latest/installation/ubuntu/



I’ll let you know once the PR has merged and the links are live!

**Comment by cloudjumpercat** [comment]
@fabianrbz I started a universal quickstart in the Kuma PR (and removed the old install docs per Charly's feedback), so the URLs I previously sent won't work. I'm hoping that between the Kubernetes and Universal quickstarts, we can use those URLs for the install landing page.

File: app/_data/install_page.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            url: 'https://hub.docker.com/r/kong/kong-gateway/tags?page=1&name=amazonlinux'
+            name: Amazon Linux (2 and 2023)
+      - name: Other
+        cards:
+          - icon: /assets/images/icons/third-party/logomark-terraform.svg
+            url: https://github.com/kong/kong-terraform-aws
+            name: AWS with Terraform
+  mesh:
+    icon: /assets/images/logos/kong-mesh-logo.svg
+    description: Grains lemonade zest cilantro cozy cinnamon oatmeal comforting pumpkin spice latte.
```
[the remark this answers is not in the document]
Comment (acgoldsmith) [reply app/_data/install_page.yml:49]: Replacing placeholder text w/ reused copy from original /install/ page + @connorckong's suggested CTA from March 4th discussion & comment in JIRA.

Need to confirm the CTA URL, currently using link from /gateway/latest/install/docker/#prerequisites

File: app/_data/install_page.yml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        cards:
+          - icon: /assets/images/icons/third-party/logomark-azure.svg
+            url: /hub/bitnami/microsoft_azure?install=oss
+            name: Kong on Microsoft Azure
+          - icon: /assets/images/icons/third-party/logomark-terraform.svg
+            url: /hub/kong-inc/kong-terraform-aws/
+            name: Kong on AWS with Terraform
+  kuma:
+    icon: /assets/images/logos/kuma-logo.svg
+    description: Grains lemonade zest cilantro cozy cinnamon oatmeal comforting pumpkin spice latte.
```
Comment (acgoldsmith) [inline app/_data/install_page.yml:125]:
```suggestion
    description: Built for hybrid and multi-cloud environments, optimized for microservices and distributed architectures. If you want to run Kuma as a part of a production-ready API platform, you can [get started with Konnect] (https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=docker-install) in under 5 minutes.
```

Comment (acgoldsmith) [reply app/_data/install_page.yml:125]: Replacing placeholder text w/ reused copy from original /install/ page + @connorckong's suggested CTA from March 4th discussion & comment in JIRA.

Need to confirm the CTA URL, currently using link from /gateway/latest/install/docker/#prerequisites

**Review by acgoldsmith — CHANGES_REQUESTED** [review]
updates to copy

File: app/_data/install_page.yml (hunk 2 above)
Comment (acgoldsmith) [reply app/_data/install_page.yml:125]: @fabianrbz , you can disregard these suggestions, they'll be obsoleted by the new install plans coming from Connor.

**Pull request closed without merging**